test/test-pages/001/expected-metadata.json

```diff
@@ -0,0 +1,5 @@
+{
+  "title": "Get your Frontend JavaScript Code Covered | Code",
+  "byline": "Nicolas Perriault —",
+  "excerpt": "width=device-width"
```

Comment:
This one shouldn't be too hard to fix*, as the html for this page is:

```html
    <meta name="description" content="Nicolas Perriault's homepage.">
    <meta name="viewport" content="width=device-width">
```
It seems the lib is simply targeting the wrong meta tag.

* famous last words